users/serializers.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib.auth.hashers import make_password
+from rest_framework import serializers
+
+from .models import User
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = [
+            "user_id",
+            "first_name",
+            "last_name",
+            "email",
+            "password",
+            "created_at",
+            "updated_at",
+            "is_active",
```

Comment:
I have omitted the keys `is_staff` and `is_superuser` from here, because i didn't think it matters for the user to actually receive this information, however if i am missing something and it does matter i can implement those values again.